### DISCUSSION TITLE
开放容器标准(OCI) 内部分享

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#234] [Announcements]
# 开放容器标准(OCI) 内部分享

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/2019/08/06/oci-intro/

**Comment by o0xmuhe** [comment]
好文，感谢分享~

**Comment by nidbCN** [comment]
非常好文章

**Comment by gabwalkerd** [comment]
写得非常好，感谢分享
